app.js

```diff
@@ -56,17 +67,28 @@ app.get('/', function(req, res) {
     connection.query('SELECT * FROM `users`', function(error, results, fields){
         if (error) {
             data = 'ClearDB is down!'
-            res.render('home', {data: data})
+            res.render(home, {data: data})
```

Comment:
The first argument in a res.render() call has to be a string which == the name of the handlebars partial you want to render minus the *.hbs extension.

Reply:
I add a route constants tab to the top of the file so that if we change the inventory route, we just have to change 1 constant and not however many res.render() routes we have in the code. 

Also, if we change the const variable name all together, we only have to refactor 1 variable (and not worry about "/inventory" vs 'inventory' in each location, etc.